stdlib-integration-tests/udp/tests/udp_tests.bal

```diff
@@ -57,7 +57,7 @@ function testConnectionlessClient() {
 
         var sendResult = socketClient->sendDatagram(datagram);
         if (sendResult is ()) {
-            log:print("Datagram was sent to the remote host.");
+            log:printInfo("Datagram was sent to the remote host.");
```

Comment:
* Isn't it better to use `log:printDebug` in test-cases ?

Reply:
here we are  logging just to keep track of what happened so it's better to use `log:printInfo` IMO.